src/applications/personalization/profile/components/direct-deposit/DirectDeposit.jsx

```diff
@@ -26,11 +28,15 @@ const DirectDeposit = () => {
       </h2>
       <DowntimeNotification
         appTitle="direct deposit"
-        render={handleDowntimeForSection('direct deposit')}
+        render={handleDowntimeForSection(
+          'direct deposit for compensation and pension',
+        )}
         dependencies={[externalServices.evss]}
       >
-        <DirectDepositContent />
+        <DirectDepositCNP />
```

Comment:
This is the main change: We don't want to hide _all_ the content on the Direct Deposit section just because EVSS is down. We only need to hide the content that's related to Direct Deposit for CNP; we can still show the info about being a victim of fraud and the info about Direct Deposit for EDU.